src/common/utils/compare.ts

```diff
@@ -365,3 +365,34 @@ export const getCompareSubheader = (region: Region): string => {
     return 'Compare counties';
   }
 };
+
+// Value maps for filters' slider components (0, 50, and 99 are numerical values required by MUI Slider).
```

Comment:
These maps were previously in Filter.tsx and CompareMain.tsx. Moved them all here